tests/service-config-schema.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
For all of the new `service-config.json.template` files, is it possible to provide this reference as `{"$schema": "<url>"}`?

Since the schema and configs are relevant for the node registry, it should be specified under the node registry source code itself instead of birdhouse-deploy.

Reply:
That's a good idea. Since the node registry source code is still in development why don't you make this an issue and we can revisit it in a later PR.

Reply:
Yes. Sounds good. I let you create it.